src/components/dashboard/sidebar/sidebar-layout.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import { navigate } from "@reach/router"
+
+import CollapseIcon from "../../../assets/img/collapse-sidebar.png"
+import ExpandIcon from "../../../assets/img/expand-sidebar.png"
+import MenuIcon from "../../../assets/img/menu-management.png"
+import SettingsIcon from "../../../assets/img/settings.png"
+
+
+let StyledLayout = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+`;
+
+let PageContent = styled.div`
+  max-width: 95%;
+  width: ${ props => props.sidebarOpen ? "87%" : "95%"};
+  transition: width 0.5s ease;
+`;
+
+let Sidebar = styled.div`
+  width: ${ props => props.sidebarOpen ? "13%" : "5%"};
```

Comment:
Would you think that it's better & easy to change pixel values for the sidebar width?